src/export/general/export.go

```diff
@@ -124,33 +132,42 @@ func (data *OverallStats) updateData() error {
 	} else {
 		return err
 	}
+
+	cpuLoad := cpuInfo.NewCPULoad()
+	err = cpuLoad.UpdateCPULoad()
+	if err != nil {
+		return err
+	}
+	data.CpuLoad = *cpuLoad
+
 	endUpdateTime := uint64(time.Now().Unix())
 	avg := uint64((startUpdateTime + endUpdateTime) / 2)
 	data.Epoch = avg
 	return nil
 }
 
-func getJSONData(iter int64, interval uint64) ([]OverallStats, error) {
+func getJSONData(iter uint32, refreshRate uint64) ([]OverallStats, error) {
 	var data []OverallStats
-	var i int64
-	stats := &OverallStats{}
+	var i uint32
+	stats := NewOverallStats()
 	for i = 0; i < iter; i++ {
 		err := stats.updateData()
 		if err != nil {
```

Comment:
Here again, since you're not using `i` anywhere except the for loop, might as well use `for i := 0; i < iter; i++`? Reduces number of lines of code is all.

Reply:
`i:=0` will infer the type of `i` as `int` and comparison won't be valid with `uint32`. Maybe I can declare it in the loop.

Reply:
Ah, I missed that. Change if you want to, this works too.

Reply:
@Gituser143 turns out I can't do [that](https://stackoverflow.com/questions/43567914/how-to-declare-variable-types-for-loop-variables-in-go/43567978).

Reply:
How about `for i := uint32(0); i < iter; i++` :P